subsys/bluetooth/host/hci_core.c

```diff
@@ -5964,6 +5989,21 @@ int bt_le_adv_stop(void)
 		return 0;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_PERIPHERAL)) {
+		conn = bt_conn_lookup_state_le(BT_ADDR_LE_NONE,
```

Comment:
You can move the scope of the `conn` variable into this branch, rather than it being global to the function (following the principle of declaring variables in the smallest possible scope)

Reply:
Yup. Moved it.